lib/dl_core_testing/dl_core_testing/dataset_builder.py

```diff
@@ -189,29 +186,23 @@ def add_source_avatar(self, source_id: str) -> SourceAvatarProxy:
         )
 
     def add_avatar_relation(
-        self,
-        left_avatar_id: str,
-        right_avatar_id: str,
-        conditions: list[BinaryCondition],
+        self, left_avatar_id: str, right_avatar_id: str, conditions: list[BinaryCondition], required: bool = False
     ) -> AvatarRelationProxy:
         relation_id = str(uuid.uuid4())
         self.ds_wrapper.add_avatar_relation(
             relation_id=relation_id,
             left_avatar_id=left_avatar_id,
             right_avatar_id=right_avatar_id,
             conditions=conditions,
+            required=required,
         )
         return AvatarRelationProxy(
             dataset_builder=self,
             relation_id=relation_id,
         )
 
     def add_avatar_relation_simple_eq(
-        self,
-        left_avatar_id: str,
-        right_avatar_id: str,
-        left_col_name: str,
-        right_col_name: str,
+        self, left_avatar_id: str, right_avatar_id: str, left_col_name: str, right_col_name: str, required: bool = False
```

Comment:
add comma in the end so it would stay multilined

Reply:
done